packages/flutter/lib/src/painting/binding.dart

```diff
@@ -70,8 +70,18 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
   ImageCache createImageCache() => ImageCache();
 
   /// Calls through to [dart:ui] with [decodedCacheRatioCap] from [ImageCache].
-  Future<ui.Codec> instantiateImageCodec(Uint8List list) {
-    return ui.instantiateImageCodec(list);
+  ///
+  /// The [cacheWidth] and [cacheHeight] parameters, when specified, indicate the
+  /// resoluton to decode the image to.
```

Comment:
unless i'm confused about how this API works, they control the size of the decoded image, not its resolution.

Reply:
I was using the phrase "resolution" a bit loosely, should probably be "size" to be accurate.

Reply:
Applicable to uses of "resolution" in `image_provider.dart` as well.